packages/core/src/models/view/filter/filter.ts

```diff
@@ -87,3 +87,30 @@ export const mergeFilter = (filter1?: IFilter, filter2?: IFilter) => {
     conjunction: 'and',
   } as IFilter;
 };
+
+export function extractDefaultFieldsFromFilters(filters: IFilterSet | null | undefined): {
```

Comment:
We need to consider the scenario where the filtering conditions include an OR. I don't think we should add all the values.